compute/managed-instances/demo/main.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python3
+from http.server import BaseHTTPRequestHandler, HTTPServer
```

Comment:
It's very unusual for us to use http.server here; almost universally we'd use Flask for a project like this in sample code and then gunicorn to serve (see the newer App Engine samples, App Engine Flex, etc.). Is there a specific reason we should use http.server? It seems like socketserver, ThreadingMixIn, etc. are boilerplate that distract from the main point.

Reply:
Okay, re-written using Flask.

Which invalidates a few other comments here, which I have marked.